src/m365/aad/commands/o365group/o365group-conversation-list.ts

```diff
@@ -0,0 +1,60 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import commands from '../../commands';
+import { Conversation } from '@microsoft/microsoft-graph-types';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import Utils from '../../../../Utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  groupId?: string;
```

Comment:
The groupId is required. Therefore you should remove the ?

Reply:
Thank you. I have removed the ?